doc/source/whatsnew/v1.5.2.rst

```diff
@@ -16,6 +16,8 @@ Fixed regressions
 - Fixed regression in :meth:`Series.replace` raising ``RecursionError`` with numeric dtype and when specifying ``value=None`` (:issue:`45725`)
 - Fixed regression in :meth:`DataFrame.plot` preventing :class:`~matplotlib.colors.Colormap` instance
   from being passed using the ``colormap`` argument if Matplotlib 3.6+ is used (:issue:`49374`)
+- Fixed regression in :meth:`.arrays.date_range` returning an invalid set of periods for CustomBusinessDay and ``start`` date with
+  timezone (:issue:`49441`)
```

Comment:
Dont think this needs to be on a newline?

Reply:
I agree, it has 159 chars for markdown its ok. I will fix it